auth/team.go

```diff
@@ -36,14 +36,15 @@ func TeamService() (authTypes.TeamService, error) {
 	}, nil
 }
 
-func (t *teamService) Create(name string, user *authTypes.User) error {
+func (t *teamService) Create(name string, tags []string, user *authTypes.User) error {
 	if user == nil {
 		return errors.New("user cannot be null")
 	}
 	name = strings.TrimSpace(name)
 	team := authTypes.Team{
 		Name:         name,
 		CreatingUser: user.Email,
+		Tags:         tags,
```

Comment:
I'd add a cleanup in `Create` and `Update` methods to trim spaces and remove duplicated tags.